Lombiq.NodeJs.Extensions/scripts/handle-error.js

```diff
@@ -4,16 +4,19 @@
 
 const os = require('os');
 
-// Treat this dependency as optional because it's not available everywhere.
-let chalk;
-try {
-    chalk = require('chalk');
-}
-catch {
-    chalk = false;
+function handleColoring(output, type, errorColoring, warningColoring) {
+    let error = type === 'error';
+  
+    if (error && typeof errorColoring === 'function') {
+      return errorColoring(output);
+    } else if (!error && typeof warningColoring === 'function') {
+      return warningColoring(output);
+    }
+
+    return output;
 }
 
-function handleErrorObjectInner(error, type, defaultCode) {
+function handleErrorObjectInner(error, type, defaultCode, errorColoring, warningColoring) {
```

Comment:
You've added two new parameters here, but don't use them anywhere.

I was just looking at the places where we would need to add the coloring callbacks, and I'm more and more convinced that this whole issue does not really make sense to pull through to the end, because it will make the code much more complicated and duplicated.

Coloring the output should be done where it currently is - when printing the output.

Please abandon this issue, close the PR, and set the value to something that more closely reflects your effort up until now. Then, resolve the issue with "Won't Do" and point to this comment.